features/steps/insights_content_service.py

```diff
@@ -63,6 +63,13 @@ def check_tags_and_groups(context):
 
 @then('rules are properly loaded')
 def check_rules_status(context):
+    '''
+    we check that if a rule is loaded than there's no error
+    and viceversa.
```

Comment:
Just a nitpick: this docstring isn't formatted propertly. Maybe it's time to add some linters for our BDD Python code